packages/rrweb/src/replay/index.ts

```diff
@@ -1085,10 +1131,17 @@ export class Replayer {
       }
       case IncrementalSource.Drag:
       case IncrementalSource.TouchMove:
-      case IncrementalSource.MouseMove:
+      case IncrementalSource.MouseMove: {
+        const pointerId =
+          'pointerId' in d && d.pointerId !== undefined
+            ? d.pointerId
+            : Object.keys(this.pointers).length;
+        if (!this.pointers[pointerId]) {
+          this.createNewTouch(pointerId);
+        }
```

Comment:
Do we clean up these nodes when movement ends?

Reply:
we have this code so far:
https://github.com/getsentry/rrweb/blob/c92bc1b4c8f0d07f17413f52e77cec79bf8fcd9c/packages/rrweb/src/replay/index.ts#L1254
but i can try cleaning it up by deleting the pointer from `pointers` as well